ansible/MANUAL_STEPS.md

```diff
@@ -479,3 +479,47 @@ After these steps are performed and the Pi's are running, Ansible can be run to
 [Setting up a Windows Host]: https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html
 [newer Ansible configuration]: https://github.com/nodejs/build/tree/master/ansible
 [stand-alone]: https://github.com/nodejs/build/tree/master/setup/windows
+
+## IBM i
+
+There isn't a system start service on IBMi -- the machine should not be
+rebooted, and after ansible is run, jenkins needs to be started with
+`jenkins-start.sh`. This will submit the job under the iojs user. If the 
+job is already running, the `jenkins-start.sh` script will not start
+another job.
+
+
+### Install open source ecosystem
+See http://ibm.biz/ibmi-rpms (see "Installation" section)
+
+### Create Nodejs user's home directory
+```
+mkdir -p /home/NODEJS
+chown -R nodejs /home/NODEJS
+```
+
+### Set global PATH and .bashrc to use Open Source Ecosystem
+Edit `/QOpenSys/etc/profile` to contain:
+```
+PATH=/QOpenSys/pkgs/bin:$PATH
+export PATH
+```
+This can be done by running the following commands from a shell:
+```
+echo 'PATH=/QOpenSys/pkgs/bin:$PATH' >> /QOpenSys/etc/profile
+echo 'export PATH' >> /QOpenSys/etc/profile
+```
+After that is completed, copy to the `.bashrc` file for the nodejs user
+`cp /QOpenSys/etc/profile /home/NODEJS/.bashrc`
+
+### Use bash as the default shell (maintainer convenience only)
+```
+yum install chsh
+chsh -s /QOpenSys/pkgs/bin/bash
+```
+
+### Install Ansible version 2.7.6
+
+Ansible 2.7.6 is required, until the resolution of [this issue](https://github.com/ansible/ansible/issues/64031).
+
+`pip3 install ansible==2.7.6`
```

Comment:
FWIW I didn't seem to run into any issues running the playbook with ansible 2.9.9. (The https://github.com/ansible/ansible/pull/69484 isn't necessary as we've used `use_backed=yum` on the relevant tasks for IBM i.)

Reply:
Oooooohh. Nice! I'll remove this bit